src/crypto/cn_turtle_hash.c

```diff
@@ -22,6 +22,8 @@
 #define INIT_SIZE_BLK          8
 #define INIT_SIZE_BYTE         (INIT_SIZE_BLK * AES_BLOCK_SIZE)
 
+extern const bool cpu_aes_enabled;
```

Comment:
Don't think you need extern here and it's value not being set anywhere methinks.

Reply:
It's using the variable set in the cn_heavy code (set in exactly one of soft, hard_arm, or hard_intel cpp files)

Reply:
LGTM. Didn't realise this was a forward declare, thought it was pulling in the heavy hash's headers forward declare.

Reply:
I tried that first, but that didn't work because this is C, not C++ (thus the extern here, and the `extern "C"` in the header).